packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/index.js

```diff
@@ -35,11 +35,16 @@ class CreateNewSandbox extends React.PureComponent {
     document.removeEventListener('keydown', this.keydownListener);
   }
 
-  createSandbox = template => {
-    this.setState({ forking: true }, () => {
+  createSandbox = (template, e) => {
```

Comment:
I think it'd be nicer to have the second argument be something like options: `{ openInNewTab: boolean }` together with a default.